libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.ts

```diff
@@ -194,7 +297,7 @@ export class FixedCardLayoutComponent implements OnInit, AfterContentInit, After
     }
 
     /** Distribute cards on window resize */
-    public updateLayout(): void {
+    updateLayout(): void {
```

Comment:
Seems like that method and others are public so the customers can use it, is it? In that case I don't see any updates for the documentation. Otherwise hide it please.

Reply:
done